woocommerce/woocommerce-android
# Add option to simulate interac payment to the Simulated Card Reader

Closes: No issue was  created for this PR


### Description

It adds a new row based on the cardReaderConfig to enable testing using the Card Reader Simulator and Interac Payments. If the Store country is Canada, the new row will be displayed once you enable the simulated card reader from the developer options screen. If the store is not based in Canada, the option is not displayed. 

### Testing instructions


1. Access a Canadian Stor
2. go to the Menu
3. Click on Developer Options
4. Enable the Simulated Card Reader
5. Make sure the Interac row is displayed
6. Enabled Interac Payments
7. create an order or a simple payment
8. complete the payment using Wisepad3 reader
9. Make sure the payment was done using Interac either by checking the logs or by checking Stripe Dashboard if you have access to that
10. Go back to the Dev Options screen and disable Interac payments
11. Create a new payment
12. Make sure the payment used Card and not interac in the logs or Stripe Dashboard
13. Finally, switch to an US store 
14. go to the dev option screen
15. Make sure the Interac row is not displayed

### Images/gif


US vs CA (splitting the videos so it does not get gigantic)

 <img src="https://user-images.githubusercontent.com/30724184/214105665-79a8d84f-8067-41f5-abe0-b5ed4ae2c9f6.gif" width="350"/> 

Interac Payment

 <img src="https://user-images.githubusercontent.com/30724184/214111891-64fe530c-2fee-408b-9f66-df48a06ab748.mp4" width="350"/>
